### PR TITLE
New language: Introducing Dutch for Neon

### PR DESCRIPTION
**Pull request by Woodehh**
**What current issue(s) from Trello/Github does this address?**
None

**What problem does this PR solve?**
The lack of the Dutch language

**How did you solve this problem?**
By translating and adding a Dutch flag to the ecosystem

**How did you make sure your solution works?**
By checking everything in line.

**Are there any special changes in the code that we should be aware of?**
No

**Is there anything else we should know?**
I had to recreate the Snapshots of the unittester by using `yarn jest --updateSnapshot` otherwise it wouldn't compile. Oh and Dutchies are cool.

- [ ] Unit tests written?

### CODE REVIEW

File: app/translations/dutch.js [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+
+  walletManagerRemoveWallet: 'Wallet verwijderen',
+
+  selectAssets: 'Select Assets',
+  priorityTransfer: 'Priority Transfer',
+
+  editRecipients: 'Edit Recipients',
+  confirmAndSend: 'Confirm & Send',
+  fee: 'Fee:',
+  sendMoreAssets: 'Send More Assets',
```
Comment (ixje) [inline app/translations/dutch.js:61]: Missing translation

File: app/translations/dutch.js [hunk 2]
```diff
@@ -0,0 +1,396 @@
...
+  transactionId: 'Transaction ID:',
+  numberofTransactionsPending:
+    '{transferCount, number} {transferCount, plural, one {Transactie} other {Transacties}} in afwachting',
+  assetRecipients:
+    'Asset {transferCount, plural, one {Ontvanger} other {Ontvangers}}',
+  confirmation: 'Confirmation',
+  confirmationRecipient:
+    '{transferCount, number} {transferCount, plural, one {Recipient} other {Recipients}}',
```
Comment (ixje) [inline app/translations/dutch.js:69]: only partially translated

File: app/translations/dutch.js [hunk 3]
```diff
@@ -0,0 +1,396 @@
...
+  'errors.request.max': `Je kunt niet meer dan 100.000.000 {asset} aanvragen.`,
+  'errors.request.min': `Je kunt niet 0 {asset} aanvragen.`,
+  'errors.network.general': 'Oops! Iets ging verkeerd.',
+  'errors.encrypt.valid': 'De private sleutel is niet geldig.',
+
+  'errors.send.balance': `Je hebt niet genoeg in je wallet om {total} {asset} te verzenden.`,
+  'errors.send.network': 'Een netwerk probleem is opgetreden.',
+  'errors.send.number': 'Aantal moet nummeriek zijn.',
+  'errors.send.fraction': 'Je knt geen gedeelte van een NEO verzenden.',
+  'errors.send.negative': `Je unt geen negatieve aantallen {asset} verzenden`,
```
Comment (ixje) [inline app/translations/dutch.js:131]:
```suggestion
  'errors.send.negative': `Je kunt geen negatieve aantallen {asset} verzenden`,
```

File: app/translations/dutch.js [hunk 4]
```diff
@@ -0,0 +1,396 @@
...
+  'errors.send.zero': `Je kunt geen 0 {asset} verzenden.`,
+  'errors.send.decimal': `Je kunt {asset} tot maximaal {decimalCount, number} verzenden.`,
+  'errors.send.invalidAddress': 'Je moet een geldig NEO adres invoeren..',
+  'errors.send.blackListed':
+    'Adres is geblacklist, dit is een bekend phising adres.',
+}
+
+const NOTIFICATIONS = {
+  'notifications.success.receivedBlockchainInfo':
+    'Laaste blockchain informatie opgevraagd.',
```
Comment (ixje) [inline app/translations/dutch.js:141]:
```suggestion
    'Laatste blockchain informatie opgevraagd.',
```

File: app/translations/dutch.js [hunk 5]
```diff
@@ -0,0 +1,396 @@
...
+  authSaved: 'OPGESLAGEN',
+  authPrivate: 'PRIVATE KEY',
+  authEncrypted: 'VERSLEUTELD',
+  authWatch: 'ALLEEN LEZEN',
+  authLedger: 'LEDGER',
+  authCreateWallet: 'Maak wallet',
+  authImportWallet: 'Importeer wallet',
+  authScanQRButton: 'Scan QR-Code',
+  authLoginButton: 'Inloggen',
+  authLedgerFirstStep: 'Verbind en ontgrenden je Ledger-apparaat',
```
Comment (ixje) [inline app/translations/dutch.js:159]:
```suggestion
  authLedgerFirstStep: 'Verbind en ontgrendel je Ledger-apparaat',
```

File: app/translations/dutch.js [hunk 6]
```diff
@@ -0,0 +1,396 @@
...
+  walletCreationWalletPasswordConfirmPlaceholder: 'Wachtwoord nogmaals',
+  walletCreationButton: 'Maak wallet',
+  walletCreatedHeader: 'Wallet aangemaakt!',
+  walletImportedHeader: 'Wallet aangemaakt!',
+  walletCreatedDisclaimer:
+    '<b>Bewaar deze details!</b> Als je deze inloggegevens kwijtraakt, heb je geen toegang meer tot je assets.',
+  privateKeyLabel: 'PRIVATE KEY',
+  encryptedKeyLabel: 'VERSLEUTELDE KEY',
+  addressLabel: 'PUBLIEK ADRES',
+  splitKeyLabel: 'GESPLITSTE SLEUTEL',
```
Comment (ixje) [inline app/translations/dutch.js:181]: inconsistent translation; either use no translation ("private key") everywhere, or full translation "privé sleutel". This is a mix of sometimes partially translated, sometimes not at all.

File: app/translations/dutch.js [hunk 7]
```diff
@@ -0,0 +1,396 @@
...
+  receiveAssetsAddressLabel: 'Jouw publieke adres',
+  receivePageLabel: 'Ontvang assets',
+  receiveYourAddressTabLabel: 'JOUW ADRES',
+  receiveCopyCodeButton: 'Copy Code Afbeelding',
+  receiveDisclaimer:
+    'Stuur alleen assets die <b>compatibel zijn met de NEO-blockchain (NEO, GAS, etc.)</b>. Het verzenden van andere assets zal resulteren in permanent verlies.',
+  receiveRequestTabAssets: 'VERZOEK ASSETS',
+  recieveWhyUseQRLabel: 'Waarom een QR-Code gebruiken??',
+  receiveQRExplanation:
+    '<p>Ooit assets naar het verkeerde adres gestuurd vanwege een foutief karakter in het wallet-adres?</p><p>Zo niet, dan heb je mazzel - maar het gebeurt veel te vaak.</p><p>Hier bij CoZ willen we ervoor zorgen dat mensen die jou betalen, het juiste adres krijgen. J kunt een QR-code genereren om assets aan te vragen, zodat ze jou kunnen helpen. </p> <p> Elke code die je genereert, bevat uw openbare wallet-adres, een aantal en een referentie - allemaal door jou ingesteld. </p>',
```
Comment (ixje) [inline app/translations/dutch.js:249]:
```suggestion
    '<p>Ooit assets naar het verkeerde adres gestuurd vanwege een foutief karakter in het wallet-adres?</p><p>Zo niet, dan heb je mazzel - maar het gebeurt veel te vaak.</p><p>Hier bij CoZ willen we ervoor zorgen dat mensen die jou betalen, het juiste adres krijgen. Je kunt een QR-code genereren om assets aan te vragen, zodat ze jou kunnen helpen. </p> <p> Elke code die je genereert, bevat je openbare wallet-adres, een aantal en een referentie - allemaal door jou ingesteld. </p>',
```
Typo + mix of informal and formal. Given that you used informal pretty much everywhere else, I suggest to keep it here as well.

File: app/translations/dutch.js [hunk 8]
```diff
@@ -0,0 +1,396 @@
...
+}
+
+const TRANSACTION_FEE = {
+  transactionFeeQuestion: 'PRIORITEER JOUW TRANSACTIE MET EEN FEE?',
+  fast: 'Snel',
+  faster: 'Sneller',
+  fastest: 'Snelst',
+  sendWithFee: 'Verstuur {itemCount, plural, one {Asset} other {Assets}} Met fee',
+  sendWithoutFee:
+    'Verstuur {itemCount, plural, one {Asset} other {Assets}} Zonder fee',
```
Comment (ixje) [inline app/translations/dutch.js:276]: "one", "other"

File: app/translations/dutch.js [hunk 9]
```diff
@@ -0,0 +1,396 @@
...
+  sendAssetCapital: 'Asset',
+  sendAssetsCapital: 'Assets',
+  sendRecipientCapital: 'ONTVANGER',
+  sendRecipientsCapital: 'ONTVANGERS',
+  sendCompleteNotification:
+    'Transactie in behandeling! Je saldo wordt automatisch bijgewerkt zodra de blockchain het heeft verwerkt.',
+  sendSelectAssets:
+    '{transferCount, number} van {maxNumberOfRecipients, number} Ontvangers',
+  sendTransferMessage:
+    '{transferCount, number} {transferCount, plural, one {sendTransfer} other {sendTransferPlural}} pending',
```
Comment (ixje) [inline app/translations/dutch.js:306]: "one", "other", "pending"

File: app/translations/dutch.js [hunk 10]
```diff
@@ -0,0 +1,396 @@
...
+  sendBroadcasting: 'Transactie uitzenden naar netwerk...',
+  sendDisclaimer:
+    'Controleer en zorg ervoor dat je de juiste gegevens hebt ingevoerd om verlies van aassets te voorkomen.',
```
Comment (ixje) [inline app/translations/dutch.js:309]:
```suggestion
    'Controleer en zorg ervoor dat je de juiste gegevens hebt ingevoerd om verlies van assets te voorkomen.',
```

File: app/translations/dutch.js [hunk 11]
```diff
@@ -0,0 +1,396 @@
...
+  networkSettingsInstructions:
+    'Beheer alle netwerkinstellingen met betrekking tot de interactie tussen Neon en de blockchain',
+  networkSettingsNodeSelectLabel: 'NODE SELECTIE',
+  networkSettingsExplorerLabel: 'BLOK VERKENNER',
+  networkSettingsCurrentLabel: 'HUIDIG NETWERK',
+  networkSettingsAutomatic: 'AUTOMATISCH',
+}
+
+const NODE_SELECTION = {
+  nodeSelectPanelHeader: 'Node Selection',
```
Comment (ixje) [inline app/translations/dutch.js:342]:
```suggestion
  nodeSelectPanelHeader: 'Node Selectie',
```

File: app/translations/dutch.js [hunk 12]
```diff
@@ -0,0 +1,396 @@
...
+  nodeSelectSelectAutomatically: 'Selecteer automatisch',
+  nodeSelectInfo: 'Top {nodeCount, number} nodes getoont',
+  nodeSelectBlockHeight: 'Blok hoogte',
+}
+
+const ENCRYPT_KEY = {
+  encryptPanelHeader: 'Beveilig een sleutel',
+  encryptInstructions: 'Kies een wachtwoord om een bestaande sleutel te beveiligen',
+  encryptStep1Label: '1) VOER DE PRIVATE KEY IN DIE JE WILT BEVEILIGEN',
+  encryptStep1Placeholder: 'Voer sleutel in',
```
Comment (ixje) [inline app/translations/dutch.js:354]: mix "private key", "sleutel", same as before. Either full translate or don't

**Review by ixje — CHANGES_REQUESTED** [review]
Thanks! A couple of minor typo's comments, other than that looks good 👍

File: app/translations/dutch.js (hunk 1 above)
Comment (Woodehh) [reply app/translations/dutch.js:61]: Missed that one. I took the liberty to change `fee` to `vergoeding` all over.

File: app/translations/dutch.js (hunk 8 above)
Comment (Woodehh) [reply app/translations/dutch.js:276]: In the german.js is the following line:

sendWithFee:
    'Sende {itemCount, plural, one {Anlagegut} other {Vermögenswerte}} mit Gebühr',

So i thought this was some sort of plural operator. Could someone confirm this?

Comment (Woodehh) [reply app/translations/dutch.js:276]: I've just tested this and the one and other are indeed the definition of 'one asset' and 'multiple assets'. Tested and worked. I've also changes asset and assets to lowercase. Seems more in style with the Dutch language.

Comment (ixje) [reply app/translations/dutch.js:276]: @comountainclimber can you share your thoughts on this? I have no idea where this is shown or if it is relevant to translate this. Thanks

File: app/translations/dutch.js (hunk 9 above)
Comment (Woodehh) [reply app/translations/dutch.js:306]: One and other are operators, pending is changed.

File: app/translations/dutch.js (hunk 6 above)
Comment (Woodehh) [reply app/translations/dutch.js:181]: I agree, I've changed as follows. Public key and private keys do not have descent translations in Dutch. So I've used public key and private key as terms: 'names' so to say. The encrypted key is now a 'Beveiligde Sleutel'.

File: app/translations/dutch.js (hunk 12 above)
Comment (Woodehh) [reply app/translations/dutch.js:354]: Check!

File: app/translations/dutch.js (hunk 2 above)
Comment (Woodehh) [reply app/translations/dutch.js:69]: Fixed

File: app/translations/dutch.js (hunk 8 above)
Comment (Woodehh) [reply app/translations/dutch.js]: Screenshots of the testing:

Schermafbeelding 2021-01-15 om 14 29 48
Schermafbeelding 2021-01-15 om 14 29 39

File: app/translations/dutch.js [hunk 13]
```diff
@@ -0,0 +1,396 @@
...
+
+const SETTINGS = {
+  settingsNetworkConfigLabel: 'NETWERK CONFIGURATIE',
+  settingCurrencyLabel: 'VALUTA',
+  settingsThemeLabel: 'THEMA',
+  settingsSoundLabel: 'GELUID',
+  settingsEncryptLink: 'BEVEILIG EEN SLEUTEL',
+  recoverWallet: 'HERSTEL WALLET',
+  settingsRecoverWalletLink: 'IMPORTEREN',
+  settingsBackUpLinkLabel: 'BACKUP WWrong ALLET',
```
Comment (ixje) [inline app/translations/dutch.js:325]:
```suggestion
  settingsBackUpLinkLabel: 'BACKUP WALLET',
```

Comment (Woodehh) [reply app/translations/dutch.js]: ty

File: app/translations/dutch.js (hunk 8 above)
Comment (comountainclimber) [reply app/translations/dutch.js]: This looks good to me (is in the send UX @ixje ) Thanks so much for this contribution @Woodehh and for reviewing @ixje !

**Review by comountainclimber — APPROVED** [review]

**Pull request merged**